### PR TITLE
LPS-196410 | Master

### PR DESCRIPTION
**Pull request by GergoMathe**
[LPS-196410](https://liferay.atlassian.net/browse/LPS-196410)

### CODE REVIEW

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:help

**Comment by GergoMathe** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase [hunk 1]
```diff
@@ -1573,4 +1573,25 @@ definition {
...
+
+			UserNavigator.gotoGeneral();
+		}
+
+		task ("When: The User opens the Password tab") {
+			UserNavigator.gotoPassword();
+		}
+
+		task ("Then: Can generate a WebDAV password by clicking the 'Generate WebDAV Password' button") {
+			Button.click(button = "Generate WebDAV Password");
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase:1593]: Can we add additional assertion steps here?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase [hunk 2]
```diff
@@ -540,4 +540,23 @@ definition {
...
+		task ("Given: A User navigates to Documents and Media in Site Menu") {
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "test-site-name");
+		}
+
+		task ("When: Clicks on the Kebab Menu and then on Access from Desktop, the WebDAV link and the notification prompt should be displayed") {
+			Portlet.gotoPortletOptions(portletOption = "Access from Desktop");
+		}
+
+		task ("Then: Log out") {
+			User.logoutPG();
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase:558]: Can we add additional assertion steps here?

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase (hunk 2 above)
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase]: Assertion steps added

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase [hunk 3]
```diff
@@ -178,6 +178,25 @@ definition {
...
+
+		task ("Given: A User navigates to Documents and Media in Site Menu") {
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "test-site-name");
+		}
+
+		task ("When: Clicks on the Kebab Menu and then on Access from Desktop, the WebDAV link and the notification prompt should be displayed") {
+			Portlet.gotoPortletOptions(portletOption = "Access from Desktop");
+		}
+
+		task ("Then: Verify that 'My Account' hyperlink is preent by clicking it") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase:195]: typo

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
See my review comments.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase [hunk 4]
```diff
@@ -349,6 +349,33 @@ definition {
 			userEmailAddress = "userea@liferay.com");
 	}
 
+	@description = "This is a use case for LPS-196410. A User can generate WebDAV password in their Account Settings (Password tab)."
```
[the remark this answers is not in the document]
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase]: Fixed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase (hunk 3 above)
Comment (GergoMathe) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FileUpload.testcase]: Fixed

File: modules/apps/account/account-test/src/testFunctional/paths/Account.path [hunk 5]
```diff
@@ -283,6 +283,11 @@
 	<td>//tr[contains(.,'${key_domainName}')]//span[contains(@class,' lfr-portal-tooltip')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>WEBDAV_PASSWORD_FIELD</td>
+	<td>//*[@id="_com_liferay_my_account_web_portlet_MyAccountPortlet_webDAVPassword"]</td>
+	<td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (GergoMathe) [reply modules/apps/account/account-test/src/testFunctional/paths/Account.path:290]: Separated

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:um:nocompile

**Comment by GergoMathe** [comment]
ci:test:security:nocompile

**Comment by GergoMathe** [comment]
ci:test:um:nocompile

**Comment by GergoMathe** [comment]
ci:test:sf

**Comment by GergoMathe** [comment]
ci:test:security:nocompile

**Comment by GergoMathe** [comment]
ci:test:um:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:stop:um

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**